woodwork/datatable.py

```diff
@@ -1013,6 +1013,9 @@ def mutual_information_dict(self, num_bins=10, nrows=None):
             nrows (int): The number of rows to sample for when determining mutual info.
                 If specified, samples the desired number of rows from the data.
                 Defaults to using all rows.
+            include_index (bool): If True, the column specified as the index will be included
+                in mutual information calculations. If False, the index column will not have
+                mutual information calculated for it. Defaults to False.
```

Comment:
Probably would be good to clarify here and in the other docstring that the index column will only be included if it is of a valid type for mutual information, since there are still circumstances where using `include_index=True` could result in the index not being present in the mutual information output.

Reply:
Good point. How about 

> If True, the column specified as the index will be included
                in mutual information calculations as long as its Logical Type supports mutual information.
                If False, the index column will not have mutual information calculated for it.
                Defaults to False.

Reply:
I think I'd flip the wording just a bit - it's more about mutual information supporting the LogicalType than the LogicalType supporting mutual information. So maybe something like this:
```
If True, the column specified as the index will be included as long as its LogicalType
is valid for mutual information calculations.
If False, the index column will not have mutual information calculated for it.
Defaults to False.
```